### PR TITLE
Radians to Degrees for multi-tag pose

### PR DESCRIPTION
**Pull request by MrRedness**
#1025

### CODE REVIEW

File: photon-client/src/components/dashboard/tabs/TargetsTab.vue [hunk 1]
```diff
@@ -102,7 +102,14 @@ const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings
         <tbody v-show="useStateStore().currentPipelineResults?.multitagResult">
           <td>{{ useStateStore().currentPipelineResults?.multitagResult?.bestTransform.x.toFixed(2) }}&nbsp;m</td>
           <td>{{ useStateStore().currentPipelineResults?.multitagResult?.bestTransform.y.toFixed(2) }}&nbsp;m</td>
-          <td>{{ useStateStore().currentPipelineResults?.multitagResult?.bestTransform.angle_z.toFixed(2) }}&deg;</td>
+          <td>
+            {{
+              (
+                useStateStore().currentPipelineResults?.multitagResult?.bestTransform.angle_z *
```
[the remark this answers is not in the document]
Comment (MrRedness) [reply photon-client/src/components/dashboard/tabs/TargetsTab.vue:108]: In what situation would multitagResult be valid but bestTransform.angle_z undefined?

[the remark this answers is not in the document]
Comment (mcm001) [reply photon-client/src/components/dashboard/tabs/TargetsTab.vue:108]: Should be never, actually? I'm actually not super familiar with how ?. works in JavaScript so I may need enlightning

[the remark this answers is not in the document]
Comment (MrRedness) [reply photon-client/src/components/dashboard/tabs/TargetsTab.vue:108]: I built this code and saw no issues with multi pnp but I can add the || 0 if you prefer

[the remark this answers is not in the document]
Comment (srimanachanta) [reply photon-client/src/components/dashboard/tabs/TargetsTab.vue:108]: Go ahead and add that to all multitarget props as all of them may be undefined

**Review by srimanachanta — APPROVED** [review]

**Pull request merged**